features/permissions-policy.yml

```diff
@@ -0,0 +1,7 @@
+name: Permissions policy
+description: The `Permissions-Policy` response header sets whether a policy-controlled feature, such as an API, may be used in a document.
+spec: https://w3c.github.io/webappsec-permissions-policy/
+caniuse: permissions-policy
+compat_features:
```

Comment:
I would also suggest moving the `allow` attribute into here, yanking it from the iframe feature.

Reply:
Fixed with b65c5e72800825f99126eea1088e9428b4affdc5.